packages/mira/tasks/amr_to_mmt.py

```diff
@@ -20,7 +20,14 @@ def main():
         mmt = model_from_json(amr)
 
         template_params = {}
+
+        cnt = 0
         for tm in mmt.templates:
+            # Sanitize
+            cnt = cnt + 1
+            if tm.name == None or tm.name == "":
+                tm.name = "generated-" + str(cnt)
```

Comment:
I'm sorry, but `cnt` is a bit to verse! Or my mind is bad. 😆 
```suggestion
        count = 0
        for tm in mmt.templates:
            # Sanitize
            count = count + 1
            if tm.name == None or tm.name == "":
                tm.name = "generated-" + str(count)
```

Reply:
done